sdk/dapp-kit/src/components/AccountDropdownMenu.tsx

```diff
@@ -67,3 +64,25 @@ export function AccountDropdownMenu({ currentAccount }: AccountDropdownMenuProps
 		</DropdownMenu.Root>
 	);
 }
+
+export function AccountDropdownMenuItem({
+	account,
+	active,
+}: {
+	account: WalletAccount;
+	active?: boolean;
+}) {
+	const { mutate: switchAccount } = useSwitchAccount();
```

Comment:
@williamrobertson13 should I move this out into the parent and pass a callback prop? Not sure if there is a lot of overhead

Reply:
there's not really any overhead, this should be fine!